crates/frontend/src/app.rs

```diff
@@ -209,7 +211,30 @@ pub fn app(app_envs: Envs) -> impl IntoView {
                                         </Layout>
                                     }
                                 }
-                            />
+                            /> 
+
+                            <Route 
+                              ssr=SsrMode::Async
+                              path="/admin/:tenant/snapshots"
```

Comment:
Should we keep snapshots in the url or config-versions as we are using config-versions for the endpoint and the table as as well?

Reply:
I think in UI it would become confusing if endpoints are config-versions and page shows snapshot.